rust/lanzatool/src/esp.rs

```diff
@@ -25,6 +60,7 @@ impl EspPaths {
         let esp_efi_fallback_dir = esp.join("EFI/BOOT");
 
         let bootspec = &generation.spec.bootspec;
+        let efi_fallback = EfiFallback::from_system_double(&generation.spec.bootspec.system);
```

Comment:
Do we support "cross installation"? I.e. installing AArch64 from x86? If so, we should also use the right lanzaboote binary depending on the target system.

Reply:
AFAIK, here's what we have (with this PR):

- Cross builds: you can build aarch64 images with SecureBoot files using this, because `nixpkgs.hostSystem` / `nixpkgs.crossSystem` is automatically doing the hard work for us. We need to check if *cross compilation* works though, but *emulated compilation* do work (tested with `binfmt`).
- Cross installation: you are on an x86 machine, you want to install a AArch64 machine from there, because your processor can become an AArch64 after reboot, this, we do not really support AFAIK? Except for system hard disks being moved around, I'm not sure, this is a classical scenario we want to support in priority.

Reply:
Now I understand better your comment, we could support `targetPlatform` if we want. I suggest to support this in the future.